PagingSample/app/src/main/java/paging/android/example/com/pagingsample/CheeseViewHolder.kt

```diff
@@ -16,27 +16,28 @@
 
 package paging.android.example.com.pagingsample
 
-import androidx.recyclerview.widget.RecyclerView
 import android.view.LayoutInflater
 import android.view.ViewGroup
 import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
 
 /**
  * A simple ViewHolder that can bind a Cheese item. It also accepts null items since the data may
  * not have been fetched before it is bound.
  */
-class CheeseViewHolder(parent :ViewGroup) : RecyclerView.ViewHolder(
-        LayoutInflater.from(parent.context).inflate(R.layout.cheese_item, parent, false)) {
-
+class CheeseViewHolder(parent: ViewGroup) : RecyclerView.ViewHolder(
+    LayoutInflater.from(parent.context).inflate(R.layout.cheese_item, parent, false)
+) {
+    var cheese: Cheese? = null
+        private set
     private val nameView = itemView.findViewById<TextView>(R.id.name)
-    var cheese : Cheese? = null
 
     /**
      * Items might be null if they are not paged in yet. PagedListAdapter will re-bind the
      * ViewHolder when Item is loaded.
      */
-    fun bindTo(cheese : Cheese?) {
-        this.cheese = cheese
-        nameView.text = cheese?.name
+    fun bindTo(item: CheeseItem?) {
```

Comment:
what about the letter? are the rows visually distinct?

suggest doing something like:

```
if ( item is CheeseViewItem.Separator) {
    nameView.text = "${item.letter} Cheeses"
    nameView.bold = true
} else {
    nameView.text = item.name
    nameView.bold = false
}
```

Reply:
Hmm this is probably more instructional, good idea.